server/controllers/users.js

```diff
@@ -223,3 +219,19 @@ export function clearUserCallSIDField(req, res) {
       res.status(500).json({ message: `Could not process request to clear user Call SID: ${err}` });
     });
 }
+
+
+export function volunteerCallback(req, res) {
+  const { id } = req.params;
+  return usersService.clearUserCallSID({ id })
+    .then((user) => {
+      if (user) {
+        res.status(200).json(user);
+      } else {
+        res.status(404).json({ message: 'Could not process request to clear user Call SID on volunteer callback' });
```

Comment:
Semantically, I think 404 depicts that the id of the user wasn't found.

Reply:
Good point, I can update that